flake.nix

```diff
@@ -24,6 +24,8 @@
     flake-compat.url = "github:nix-community/flake-compat";
     flake-parts.inputs.nixpkgs-lib.follows = "nixpkgs";
     flake-parts.url = "github:hercules-ci/flake-parts";
+    hydra.flake = false;
+    hydra.url = "github:qowoz/hydra/community";
```

Comment:
Should we just ask for commit access and get those commits in upstream? 
We can probably even test them on the actual hydra?

Reply:
Almost all of them have been deploy deployed on hydra.nixos.org for a while and are in various upstream branches / PRs.

Reply:
cc @zimbatm @Ericson2313 @dasJ

Reply:
@zowoq even more a reason to just merge them into the hydra branch. I ask for access.

Reply:
I'm also looking at picking the lix patches that replace hydra-eval-jobs with nix-eval-jobs. Would need to pick a couple their patches for nix-eval-jobs as well.

Reply:
I am also interested in having those patches in my nix-eval-jobs to make it work with hydra.

Reply:
I'd also like some more energy put into that hydra than various forks sprawling. But I understand it's hard since it sometimes seems nobody's really watching there.

Reply:
I still want to do CA things for hydra, and newer build protocol, but I have to finish the Mesonification first.

Reply:
@Ericson2314 mesonification of nix itself?

Reply:
> I'm also looking at picking the lix patches that replace hydra-eval-jobs with nix-eval-jobs.

Seems this will require a bit more work than just picking the patches so I'll leave it for now.

Reply:
@zowoq if would look at this, would you be ok, if we run the result on nix-community infra?

Reply:
I'm happy to pick patches from a upstream PR if that is what you mean?

Reply:
I mean, I would need to test patches as well in a non-trivial setup and do fixes as needed. Local testing of demo instances only take you so far.

Reply:
For that I'd suggest deploying another hydra instance on build01.

Reply:
I was hoping we could test hydra changes together, but in that case I could also look for a different project where I can deploy a hydra.